src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" data-qa="big">
+      <fieldset class="block big">
+        <label class="searchfield big">
+          <img 
+            src="images/Search.svg" 
+            alt="Search" 
+            class="big__pic"
+          >
+          <input 
+            type="search" 
+            class="search__bar big" 
```

Comment:
checklist p.2

You can remove `big` class here and set all the styles to the `search__bar`